python/ray/tune/ray_trial_executor.py

```diff
@@ -437,7 +503,9 @@ def start_trial(self, trial, checkpoint=None, train=True) -> bool:
             time.sleep(2)
             error_msg = traceback.format_exc()
             self._stop_trial(trial, error=True, error_msg=error_msg)
-            return False
+            # Return True here so we don't try to start a new trial
+            # in the main event loop
+            return True
```

Comment:
this is kind of weird (and changes the semantics of `start_trial`?)

Reply:
Hm right, I found another solution that makes more sense.